chia/consensus/blockchain.py

```diff
@@ -218,18 +222,18 @@ async def receive_block(
 
         genesis: bool = block.height == 0
         if self.contains_block(block.header_hash):
-            return ReceiveBlockResult.ALREADY_HAVE_BLOCK, None, None, ([], {})
+            return ReceiveBlockResult.ALREADY_HAVE_BLOCK, None, None, ([], {}), {}
 
         if not self.contains_block(block.prev_header_hash) and not genesis:
-            return (ReceiveBlockResult.DISCONNECTED_BLOCK, Err.INVALID_PREV_BLOCK_HASH, None, ([], {}))
+            return ReceiveBlockResult.DISCONNECTED_BLOCK, Err.INVALID_PREV_BLOCK_HASH, None, ([], {}), {}
 
         if not genesis and (self.block_record(block.prev_header_hash).height + 1) != block.height:
-            return ReceiveBlockResult.INVALID_BLOCK, Err.INVALID_HEIGHT, None, ([], {})
+            return ReceiveBlockResult.INVALID_BLOCK, Err.INVALID_HEIGHT, None, ([], {}), {}
```

Comment:
I thought this return type was beyond unwieldy to begin with. I really don't think we should make this more complicated. If you need to return more stuff, I think we better give the return type a name